fe2-android/app/src/test/unit/java/com/github/dedis/popstellar/utility/handler/RollCallHandlerTest.java

```diff
@@ -68,7 +67,7 @@ public void setup() throws GeneralSecurityException, IOException, KeyException {
     lenient().when(keyManager.getMainPublicKey()).thenReturn(SENDER);
     lenient().when(keyManager.getValidPoPToken(any(), any())).thenReturn(POP_TOKEN);
 
-    when(messageSender.subscribe(any())).then(args -> Completable.complete());
+    lenient().when(messageSender.subscribe(any())).then(args -> Completable.complete());
```

Comment:
Why the use of lenient ?

Reply:
By default, Mockito makes sure that mocks are actually used in tests. Using lenient disables this check.

This caused issues because every test do not trigger the subscribe function.